INSTALL

```diff
@@ -162,20 +185,20 @@ almost always okay.  It is normal for some things to be "NOT found",
 since Configure often searches for many different ways of performing
 the same function.
 
-At any Configure prompt, you can type  &-d  and Configure will use the
+At any Configure prompt, you can type C<&-d>  and Configure will use the
 defaults from then on.
 
 After it runs, Configure will perform variable substitution on all the
-*.SH files and offer to run make depend.
+*.SH files and offer to run S<C<make depend>>.
```

Comment:
Just tested out this construct in an HTML renderer (never used S before) and `C<S<foo bar>>` successfully applies the nowrap style but `S<C<foo bar>>` does not due to the nesting of span and code elements. Either seems to work for commandline perldoc.

Reply:
That's good to know, and we should document this issue.  It is a problem when the C<> is for just a subset of what needs to be on the same line.  It also needs to be documented that for links to the same pod, you should use C<L/<...>> vs on a different pod, L<C<...>>